src/migrations/20201130184519-rename-user-title-column.js

```diff
@@ -0,0 +1,50 @@
+const titles = [
+  'Program Specialist',
+  'Early Childhood Specialist',
+  'Grantee Specialist',
+  'Family Engagement Specialist',
+  'Health Specialist',
+  'Systems Specialist',
+];
+
+const roles = [
+  'Regional Program Manager',
+  'COR',
+  'Supervisory Program Specialist',
+  'Program Specialist',
+  'Grants Specialist',
+  'Central Office',
+  'TTAC',
+  'Admin. Assistant',
+  'Early Childhood Manager',
+  'Early Childhood Specialist',
+  'Family Engagement Specialist',
+  'Grantee Specialist Manager',
+  'Grantee Specialist',
+  'Health Specialist',
+  'System Specialist',
+];
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    queryInterface.removeColumn('Users', 'title');
+    queryInterface.addColumn(
```

Comment:
Can we add a transaction if there are multiple statements in one migration? The main reason is that in a case of one statement failing and other succeeding, potentially leaves us in an undesired state.